interface/main/finder/multi_patients_finder.php

```diff
@@ -231,7 +231,10 @@
 $('#add-to-list').on('click', function (e) {
     e.preventDefault();
 
-    if($('#by-name').val() == '')return;
+    if($('#by-name').val() == '') {
+        alert("<?php echo xls("You must provide a patient name or id to add to the list"); ?>");
```

Comment:
use xlj instead (note the outside quotes are also removed) (this function basically uses json_encode to escape in js context which adds the quotes):
```
alert(<?php echo xlj("You must provide a patient name or id to add to the list"); ?>);
```